ifrs17/Import/Importers.ipynb

```diff
@@ -357,9 +355,57 @@
         {
             "cell_type": "markdown",
             "source": [
-                "## Get Args and commit Partition",
+                "## Create Partition",
+                "\n",
+                "\nThese are the methods used to create the partition if not already existing in the DataSource. "
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public async Task SetPartitionAsync<IPartition>(ImportArgs args, params IDataSource[] dataSources)",
```

Comment:
I don't get this name. SetPartitionAsync, bit you are updating/committing, right?

Reply:
I think it is important we transparently name the method that commit to datasource

Reply:
"Set" is a bit confusing because this method is not setting the partition.  
What do you propose? Does "UploadPartitionAsync" read better? 
@dtrzesniak @dcolleoni

Reply:
Yes, something along these lines... Set is confusing, as you say, as it is not setting.
Upload is ok, as I think we also use this name in the same context as here.

Reply:
Ok, and what about CommitPartitionAsync? I think this is the best name because it is consistent with the method GetArgsAndCommitPartitionAsync. How do you like it?

Reply:
It's fine with me.